### ISSUE TITLE
GNOME Gitlab downloading failed

### ISSUE THREAD

**Issue by AnotherOneAckap** [PerlAlien/Alien-Libxml2#37]
Hi there! First of all, thank you for your awesome work.

I got problem installing v0.19, here goes the build log
```
Building Alien-Libxml2-0.19
Distribution opts in x_use_unsafe_inc: 0
cp lib/Alien/Libxml2.pm blib/lib/Alien/Libxml2.pm
"/usr/bin/perl" -MAlien::Build::MM=cmd -e prefix site /conf.ontico.pro/local/lib/perl5/x86_64-linux-gnu-thread-multi /conf.ontico.pro/local/lib/perl5/x86_64-linux-gnu-thread-multi /conf.ontico.pro/local/lib/perl5/x86_64-linux-gnu-thread-multi
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::CommandLine
main> prefix /conf.ontico.pro/local/lib/perl5/x86_64-linux-gnu-thread-multi/auto/share/dist/Alien-Libxml2
"/usr/bin/perl" -MAlien::Build::MM=cmd -e version 0.19
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::CommandLine
"/usr/bin/perl" -MAlien::Build::MM=cmd -e download
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::CommandLine
error fetching https://gitlab.gnome.org/api/v4/projects/GNOME%2Flibxml2/releases: 599 Internal Exception at /conf.ontico.pro/local/lib/perl5/Alien/Build/Plugin/Fetch/HTTPTiny.pm line 106.
Alien::Build::Plugin::Fetch::HTTPTiny> 599 Internal Exception fetching https://gitlab.gnome.org/api/v4/projects/GNOME%2Flibxml2/releases
Alien::Build::Plugin::Fetch::HTTPTiny> exception: Could not connect to 'gitlab.gnome.org:443': Connection timed out
make: *** [_alien/mm/download] Error 110
```

Looks like gitlab.gnome.org caused installation fail. I tried to open it in browser and got 404. 
But now it all looks good.

May be we should bundle libxml into this module, so deploying could be more independent of outer services?
Or are there another options?

**Comment by plicease** [comment]
Unfortunately bundling is problematic for a number of reasons.  The bundled version gets out of date, it bloats CPAN, it's potentially more complicated legally for the cpan author, it's more complicated for system integrators (Linux distros). Even if you do bundle the module, cpan could be down and you might not be able to fetch `Alien::Libxml2` or `XML::LibXML` at all.

TL;DR is the alternative is to install the system version of `libxml2`.

**Comment by AnotherOneAckap** [comment]
Got it! Thank you for your answer, installing system libxml2 looks acceptable for me.
